packages/shared-ux/button_toolbar/src/toolbar/toolbar.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('<Toolbar />', () => {
     const children = { primaryButton };
     const component = mountWithIntl(<Toolbar children={children} />);
 
-    expect(component).toMatchSnapshot();
+    expect(component.render()).toMatchSnapshot();
```

Comment:
Why is this necessary?

Reply:
This cuts down on the size of the snapshot itself. Emotion can bloat some of the snapshots (to the point where they're even cut off in console). This is just to reduce the size and churn of the snapshot.